test/performance/odp_crypto.c

```diff
@@ -820,6 +822,12 @@ run_measure_one(crypto_args_t *cargs,
 				}
 				packets_sent += rc;
 				packets_received++;
+				if (odp_unlikely(odp_crypto_result(&result, out_pkt) != 0) ||
+				    odp_unlikely(!result.ok)) {
+					ODPH_ERR("Crypto operation failed\n");
```

Comment:
Could be useful to print `cipher_status`/`auth_status` as well here (and below).

Reply:
I did not bother to do that since the status is a bit cumbersome (there are 4 separate status fields) and the status codes are currently not very useful (almost always you get either ALG_ERR_DATA_SIZE or ALG_ERR_ICV_CHECK). I can add the printout if you would like to have it anyway.

Reply:
I don't know, it could be useful for some implementations. Up to you.

Reply:
I decided not to add the status printout in this PR since I actually would like to change the crypto API to simplify the status indication.